docs/deployment/deploy-without-docker/Deploying.md

```diff
@@ -1,131 +1,29 @@
 # Deploying the Web Application
 
-## Prepare the global configuration file
+## Before running cbioportal backend
 
-The portal is configured using a global configuration file, `application.properties`. An example file is available in the `src/main/resources` folder. Use it as a template to create your own:
-
-```
-cd src/main/resources
-cp application.properties.EXAMPLE $HOME/cbioportal/application.properties
-```
+You will need to update the src/main/resources/applications.properties to include your DB connection information.
 
 For more information about the `application.properties` file, see the [reference](/deployment/customization/Customizing-your-instance-of-cBioPortal.md) page.
 
-Several scripts of cBioPortal use this `application.properties` file to get info like db connection parameters. You can indicate the folder where this file is with an environment variable:
-
-```
-export PORTAL_HOME=$HOME/cbioportal
-```
-
-if your properties file is at `PORTAL_HOME/application.properties`
-
-## Run cBioPortal Session Service
-
-The cBioPortal app requires [session service](/Architecture-Overview.md#session-service). For instructions on how to run this without Docker see https://github.com/cBioPortal/session-service#run-without-docker. Once this is working, update the properties file:
-
-```bash
-# session-service url: http://[host]:[port]/[session_service_app]/api/sessions/[portal_instance]/
-# example session-service url: http://localhost:8080/session_service/api/sessions/public_portal/
-# see: https://github.com/cBioPortal/session-service
-session.service.url=
-```
-
 ## Run the cbioportal backend
 
-To run the app we use `webapp-runner`. It's a command line version of Tomcat provided by [Heroku](https://github.com/jsimone/webapp-runner). All parameters can be seen with:
+To run the backend execute the following commabd
 
 ```
-java -jar portal/target/dependency/webapp-runner.jar --help
+java -jar target/cbioportal-exec.jar
 ```
 
-This runs the app in the foreground. If a port is already in use it will raise an error mentioning that. To change the port use the `--port` flag.
```

Comment:
maybe update this to reflect the new `port` parameter? This still seems like a common issue people could run into